torch/serialization.py

```diff
@@ -429,6 +430,21 @@ def load(f, map_location=None, pickle_module=pickle, **pickle_load_args):
             f.close()
 
 
+def _get_layout(name, _cache={}):
```

Comment:
If `torch.sparse_coo` would be serialized as `sparse_coo`, the `_get_layout` function would not be needed as it is as a simple `lambda name: getattr(torch, num)` would be sufficient.
However, serialization as `sparse_coo` would require string processing, this actually contains more operations than the current approach. I tried other approaches (attaching a name attribute to torch.layout instances, for instance) to get the layout name when serializing but these approaches were not much cleaner.

Reply:
I chose to serialize the full name `torch.sparse_coo` because it is readily available (via applying `str`) and there is a chance that if somebody will implement a new layout instance without the `torch` prefix then the current approach would still work while the stripping the `torch.` prefix approach might break.